chart/tests/test_flower.py

```diff
@@ -358,18 +358,18 @@ def test_defaults(self):
         "ports, expected_ports",
         [
             (
-                [{"name": "{{ .Release.Name }}", "protocol": "UDP", "port": "{{ .Values.ports.flowerUI }}"}],
```

Comment:
In [NetworkPolicyPort](https://github.com/yannh/kubernetes-json-schema/blob/master/v1.21.0/serviceport-v1.json), `name` field is illegal.